### PR TITLE
patch: Expose styling and zoom from cytoscape

### PR DESCRIPTION
**Pull request by toftalisk**
To ease the creation of client apps we expose some of cytoscape's
functionality

### CODE REVIEW

File: src/components/sparqlGraph/SparqlGraph.tsx [hunk 1]
```diff
@@ -8,8 +8,9 @@ import { ElementDefinition } from 'cytoscape';
 import CytoscapeComponent from 'react-cytoscapejs';
 import { RdfIndividual, RdfSelection, RdfTriple } from '../../models';
 import { rdfObjectKey, rdfPredicateKey, rdfSubjectKey } from './cytoscapeDataKeys';
+import { defaultStyling as defaultUiConfig } from '../../models/uiConfig';
```
Comment (lorentzfb) [inline src/components/sparqlGraph/SparqlGraph.tsx:11]: Rename defaultStyling to defaultUiConfig

File: src/models/uiConfig.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export interface UiConfig {
+	css?: React.CSSProperties;
+	maxZoom?: number;
+	minZoom?: number;
+	zoom?: number;
+	zoomingEnabled?: boolean;
+}
+
+export const defaultStyling: UiConfig = {
```
Comment (lorentzfb) [inline src/models/uiConfig.ts:9]: Rename object to "defaultUiConfig"

**Review by lorentzfb — CHANGES_REQUESTED** [review]

File: src/components/sparqlGraph/SparqlGraph.tsx [hunk 3]
```diff
@@ -8,8 +8,9 @@ import { ElementDefinition } from 'cytoscape';
...
 
-export const SparqlGraph = ({ turtleString, layoutName, onElementsSelected }: SparqlGraphProps) => {
+export const SparqlGraph = ({ turtleString, layoutName, uiConfig, onElementsSelected }: SparqlGraphProps) => {
```
Comment (DzianisMakeichyk) [inline src/components/sparqlGraph/SparqlGraph.tsx:13]:
```suggestion
export const SparqlGraph = ({ turtleString, layoutName, styles, onElementsSelected }: SparqlGraphProps) => {
```
mb `styles`? :)

File: src/models/uiConfig.ts (hunk 2 above)
Comment (DzianisMakeichyk) [inline src/models/uiConfig.ts:9]: I think if its model you need remove `defaultStyling` from here to [SparqlGraph.tsx](https://github.com/equinor/sparql-graph/pull/64/files#diff-e9c03e956a8896783efb4d9a0cd21ee3da2bd5ed24c6ac145bc504450c0130eb) and also add [uiConfig.ts](https://github.com/equinor/sparql-graph/pull/64/files#diff-d2503062c0792880e2924eb9d9b67187770cb624da60c2d5097a4bc9d595163b) to `index.ts`

File: src/components/sparqlGraph/SparqlGraph.tsx (hunk 1 above)
Comment (DzianisMakeichyk) [inline src/components/sparqlGraph/SparqlGraph.tsx:11]: forgot to add to `index.ts` :)

File: src/models/uiConfig.ts (hunk 2 above)
Comment (lorentzfb) [reply src/models/uiConfig.ts:9]: Agree. Maybe defaultUiConfig.ts?

File: src/components/sparqlGraph/SparqlGraph.tsx (hunk 3 above)
Comment (lorentzfb) [reply src/components/sparqlGraph/SparqlGraph.tsx:13]: The object does not only contain styling, but stuff like zoom etc.

File: src/components/sparqlGraph/SparqlGraph.types.ts [hunk 4]
```diff
@@ -13,3 +14,11 @@ export type LayoutWrapper = {
 	name: string;
 	layout: LayoutOptions;
 };
+
+export interface UiConfig {
```
Comment (DzianisMakeichyk) [inline src/components/sparqlGraph/SparqlGraph.types.ts:18]:
```suggestion
export interface UiConfigProps {
```

:)

**Review by DzianisMakeichyk — COMMENTED** [review]
Small thing :)

**Review by lorentzfb — APPROVED** [review]

**Pull request merged**